lnd.go

```diff
@@ -170,14 +170,18 @@ type ListenerWithSignal struct {
 
 	// Ready will be closed by the server listening on Listener.
 	Ready chan struct{}
+
+	// MacChan is an optional way to pass the admin macaroon to the program
+	// that started lnd.
+	MacChan chan []byte
 }
 
 // ListenerCfg is a wrapper around custom listeners that can be passed to lnd
 // when calling its main method.
 type ListenerCfg struct {
-	// RPCListener can be set to the listener to use for the RPC server. If
-	// nil a regular network listener will be created.
-	RPCListener *ListenerWithSignal
+	// RPCListeners can be set to the listeners to use for the RPC server.
+	// If nil a regular network listener will be created.
```

Comment:
Should probably change this to `if empty` and use `if len(lisCfg.RPCListeners) > 0 {` below.